packages/datadog-plugin-fetch/test/index.spec.js

```diff
@@ -0,0 +1,714 @@
+'use strict'
+
+const getPort = require('get-port')
+const agent = require('../../dd-trace/test/plugins/agent')
+const tags = require('../../../ext/tags')
+const { expect } = require('chai')
+const { storage } = require('../../datadog-core')
+const { ERROR_MESSAGE, ERROR_TYPE, ERROR_STACK } = require('../../dd-trace/src/constants')
+
+const HTTP_REQUEST_HEADERS = tags.HTTP_REQUEST_HEADERS
+const HTTP_RESPONSE_HEADERS = tags.HTTP_RESPONSE_HEADERS
+
+const describe = globalThis.fetch ? globalThis.describe : globalThis.describe.skip
+
+describe('Plugin', () => {
+  let express
+  let fetch
+  let appListener
+
+  describe('fetch', () => {
+    function server (app, port, listener) {
+      const server = require('http').createServer(app)
+      server.listen(port, 'localhost', listener)
+      return server
+    }
+
+    beforeEach(() => {
+      appListener = null
+    })
+
+    afterEach(() => {
+      if (appListener) {
+        appListener.close()
+      }
+      return agent.close({ ritmReset: false })
+    })
+
+    describe('without configuration', () => {
+      beforeEach(() => {
+        return agent.load('fetch')
+          .then(() => {
+            express = require('express')
+            fetch = globalThis.fetch
+          })
+      })
+
+      it('should do automatic instrumentation', done => {
+        const app = express()
+        app.get('/user', (req, res) => {
+          res.status(200).send()
+        })
+        getPort().then(port => {
+          agent
+            .use(traces => {
+              expect(traces[0][0]).to.have.property('service', 'test-http-client')
```

Comment:
This is different between version. Can the change be done in a backward-compatible way to avoid merge conflicts?

Reply:
I added #3282 to the release which remove the difference between branches.